### DISCUSSION TITLE
Cloudflare Turnstile or Google reCaptcha

### DISCUSSION THREAD

**Discussion by xKeRnx** [givanz/Vvveb#55] [Ideas]
Whould be nice to have Cloudflare Turnstile or Google reCaptcha for Login, SignUp, Password forget.

**Comment by givanz** [comment]
Thanks for the suggestion.

A captcha plugin is included in the latest update with support for Turnstile for user forms and comment/review forms.

In the following updates support for reCaptcha will also be added along with more options like enable the captcha only on certain forms.
